core/common/src/main/java/alluxio/metrics/MetricsSystem.java

```diff
@@ -237,4 +254,23 @@ public void stop() {
   public MetricRegistry getMetricRegistry() {
     return mMetricRegistry;
   }
+
+  /**
+   * Util function to remove get the metrics name without instance and host.
+   * @param metricsName the long metrics name with instance and host name
+   * @return the metrics name without instance and host name
+   */
+  public static String stripInstanceAndHost(String metricsName) {
+    String[] pieces = metricsName.split("\\.");
```

Comment:
Oh, the current implementation is just simpler. Unless you are worried about perfomance, i would just keep this. 
The alternative you suggested will result in code like this.

id 1 = metricsName.indexOf(".")
substr = metricsName.substring(id1 + 1, ..)
id2 = substr.indexOf(".")
return substr.substring(...)